tldr.py

```diff
@@ -197,7 +204,11 @@ def get_page(command, remote=None, platform=None):
 
 def colors_of(key):
     env_key = 'TLDR_COLOR_%s' % key.upper()
-    values = os.environ.get(env_key, '').strip() or DEFAULT_COLORS[key]
+    user_value = os.environ.get(env_key, '').strip()
+    if user_value and user_value not in ACCEPTED_COLORS:
+        # you can put a warning statement here, but it will print a lot
```

Comment:
this means that every time tldr prints a colored line, the warning will print as well, not just for the overall command.

Reply:
I don't like comments in code that read as messages from one developer to another. I'd rather use a voice consistent with a collaboratively developed project, e.g. "We don't print a warning here because it would print multiple times per page if the provided color is unsupported". WDYT?